src/test/javascript/e2e/entities/merchant.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+describe('Merchant e2e test', function () {
+
+    var username = element(by.id('username'));
+    var password = element(by.id('password'));
+    var entityMenu = element(by.id('entity-menu'));
+    var accountMenu = element(by.id('account-menu'));
+    var login = element(by.id('login'));
+    var logout = element(by.id('logout'));
+
+    beforeAll(function () {
+        browser.get('/');
+
+        accountMenu.click();
+        login.click();
+
+        username.sendKeys('admin');
+        password.sendKeys('admin');
+        element(by.css('button[type=submit]')).click();
+    });
+
+    it('should load Merchants', function () {
+        entityMenu.click();
+        element.all(by.css('[ui-sref="merchant"]')).first().click().then(function() {
+            element.all(by.css('h2')).first().getAttribute('data-translate').then(function (value) {
+                expect(value).toMatch(/yaldayGatewayApp.merchant.home.title/);
```

Comment:
This is more auto generated JHipster Bliss :)
I believe that this asserts that the value is now the merchants page only i.e. does not load any merchants